bot/constants.py

```diff
@@ -13,3 +14,12 @@
 DISCORD_BOOST_LVL3_CAP = 30
 
 EMBED_INFO_COLOR = 0x1E90FF
+
+# Database
+DB_FILE_PATH = os.getenv("DB_FILE_PATH")
+#absolute path to the init script
+DB_INIT_SCRIPT = os.path.join(os.path.dirname(os.path.abspath(__file__)), "persistence/resources/init_db.sql")
+
+# Database Queries
```

Comment:
I'm not fully convinced that we will need **the exact same** queries multiple times in our code and I'm therfore unsure if we should store them as constants.

Reply:
It makes it easier to manage and IMO to read. I will use a dedicated query constants file in the persistence module